config-controller/config/crd/bases/config.stackrox.io_securitypolicies.yaml

```diff
@@ -0,0 +1,200 @@
+                type: string
+              disabled:
+                type: boolean
+              enforcementActions:
+                enum:
+                - UNSET_ENFORCEMENT
+                - SCALE_TO_ZERO_ENFORCEMENT
+                - UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT
+                - KILL_POD_ENFORCEMENT
+                - FAIL_BUILD_ENFORCEMENT
+                - FAIL_KUBE_REQUEST_ENFORCEMENT
+                - FAIL_DEPLOYMENT_CREATE_ENFORCEMENT
+                - FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT
+                items:
+                  type: string
+                type: array
+              exclusions:
+                items:
+                  properties:
+                    deployment:
+                      properties:
+                        name:
+                          type: string
+                        scope:
+                          properties:
+                            cluster:
+                              type: string
+                            label:
+                              properties:
+                                key:
+                                  type: string
+                                value:
+                                  type: string
+                              type: object
+                            namespace:
+                              type: string
+                          type: object
+                      type: object
+                    image:
+                      properties:
+                        name:
+                          type: string
+                      type: object
+                    name:
+                      type: string
+                  type: object
+                type: array
+              isDefault:
+                type: boolean
+              lifecycleStages:
+                items:
+                  enum:
+                  - DEPLOY
+                  - BUILD
+                  - RUNTIME
+                  type: string
+                minItems: 1
+                type: array
+              mitreAttackVectors:
+                items:
+                  properties:
+                    tactic:
+                      type: string
+                    techniques:
+                      items:
+                        type: string
+                      type: array
+                  type: object
+                type: array
+              mitreVectorsLocked:
+                type: boolean
+              name:
+                type: string
+              notifiers:
+                items:
+                  type: string
+                type: array
+              policySections:
+                items:
+                  properties:
+                    policyGroups:
+                      items:
+                        properties:
+                          booleanOperator:
+                            enum:
+                            - OR
+                            - AND `json:"//,omitempty"`
```

Comment:
I am new to CRD. why do we need to have the json tag here? Are we defaulting it to AND?

Reply:
Definitely a copy/paste error: d78c62fd4d8ee96ae0bb513f0aaaa4db830dd096